### DISCUSSION TITLE
Solana NFT Collection (EyY2iAeJc4hjgMTdiQERfL3AwKThSDRpExqJsvB1MsWS) is hidden by default

### DISCUSSION THREAD

**Discussion by soldevcm** [phantom/docs#128] [Help & Support] [answered]
### Summary

I am a developer launching a new collection for the Oddkey brand (https://oddkey.com/). We have launched several Solana collections for the brand and never had a collection hidden by default in wallets. We were testing the mint today and noticed the NFTs are hidden. As we have the public launch tomorrow we are trying to understand what we can do to solve the issue. 

The collection ID is: EyY2iAeJc4hjgMTdiQERfL3AwKThSDRpExqJsvB1MsWS

### Steps to Reproduce

Collection ID of hidden collection: EyY2iAeJc4hjgMTdiQERfL3AwKThSDRpExqJsvB1MsWS

NFT Token address from our test: HwUWbTMHUTd1gag8h9qP31NDRZSMH7Sg93aYNvCXZJVb

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @soldevcm, I checked a few wallets with this token and it does appear to be showing up in the normal collections. Not hidden. Do you have a specific wallet where it is showing up hidden? How many reports have you had?
